<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CcFontConverter</class>
 <widget class="QMainWindow" name="CcFontConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CcFontConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="guiButtonFont">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Font</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guiEditFonts">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="guiLabelSizes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sizes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guiEditSizes">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="guiLabelSource">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guiButtonGenerate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="guiEditSources">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="guiEditPrefix">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="guiLabelPrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefix</string>
    </property>
   </widget>
   <widget class="QLabel" name="guiLabelName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[[Prefix]_](Font)_(Size)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="guiButtonGenerateFiles">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guiButtonFontFile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Font File</string>
    </property>
   </widget>
   <widget class="QToolButton" name="guiButtonFontClean">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="guiButtonSizeClean">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
